src/types/commands.ts

```diff
@@ -119,6 +119,25 @@ export const invalidateEvaluationCommands = new Set<CommandTypes>([
   "UPDATE_LOCALE",
 ]);
 
+const invalidateTableStyleCommands = [
+  "HIDE_COLUMNS_ROWS",
+  "UNHIDE_COLUMNS_ROWS",
+  "UNFOLD_HEADER_GROUP",
+  "FOLD_HEADER_GROUP",
+  "FOLD_ALL_HEADER_GROUPS",
+  "UNFOLD_ALL_HEADER_GROUPS",
+  "CREATE_TABLE",
+  "UPDATE_TABLE",
+  "UPDATE_FILTER",
+] as const;
+export const invalidateTableStyleCommandsSet = new Set<CommandTypes>(invalidateTableStyleCommands);
```

Comment:
Is there a reason to declare this in this way instead of directly declaring the set from the Array (probably a useless comments, but it would keep the same declaration style as the others set)

Reply:
Also, it doesn't need to be exported for the moment, thanks to your helper function :)